R/import_data.R

```diff
@@ -1,3 +1,58 @@
+#' Función de conveniencia para hacer peticiones a SIVIGILA
+#' @param url La dirección HTTP desde donde descargar la información
+#' @return Si la petición es exitosa, retorna una respuesta HTTP.
+#' De lo contrario arroja un mensaje de error explicando el problema
+#' y finaliza la ejecución del programa
+#' @examples
+#' \dontrun{
+#' query_event_year_path <- config::get(file =
+#'            system.file("extdata",
+#'            "config.yml",
+#'            package = "sivirep"),
+#'            "query_diseases_by_year_path")
+#' make_request(query_event_year_path)
+#' }
+#' @noRd
+
+make_request <- function(url) {
```

Comment:
Hi Jaime, is it possible to change the function's name to Spanish? I also had to change the scraping function to Spanish for the package's format.

Reply:
Done